src/boot/ksdk1.1.0/warp-kl03-ksdk1.1-boot.c

```diff
@@ -60,20 +60,21 @@
 */
 #include "devBMX055.h"
 //#include "devADXL362.h"
-#include "devMMA8451Q.h"
+//#include "devMMA8451Q.h"

```

Comment:
This breaks the default setup of master.

Reply:
I'm quite low on space, I commented out devMAG3110.h instead